lib/simplifyExpression/fractionsSearch/divideByGCD.js

```diff
@@ -50,6 +50,7 @@ function divideByGCD(fraction) {
     '/', [intermediateNumerator, intermediateDenominator]);
   const status = Node.Status.nodeChanged(
     ChangeTypes.FIND_GCD, fraction, intermediateFraction);
+  status.newNode = Node.Status.resetChangeGroups(status.newNode);
```

Comment:
Sweet, that makes sense and it worked! Here's what it looks like now:
![image](https://user-images.githubusercontent.com/1062275/28047251-bf35fcae-65b6-11e7-80d4-0d2212a1158c.png)
(Don't pay too much attention to the text around the substeps .. we're actually getting rid of that soon ... which is a whole other conversation)